From c9a704b1322aec5f699e2962368bbffd286eb010 Mon Sep 17 00:00:00 2001
From: Simi Pallipurath <simi.pallipurath@arm.com>
Date: Thu, 14 Nov 2024 10:12:33 +0000
Subject: Define picocrt_machines for AArch32 builds as well as 64.

Our cmake build command for picolibc unconditionally includes
`-Dtest-machine=...`, which depends on this being defined.
---
 picocrt/machine/arm/meson.build | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/picocrt/machine/arm/meson.build b/picocrt/machine/arm/meson.build
index 808d691a5..ecc9a485b 100644
--- a/picocrt/machine/arm/meson.build
+++ b/picocrt/machine/arm/meson.build
@@ -33,3 +33,10 @@
 # OF THE POSSIBILITY OF SUCH DAMAGE.
 #
 src_picocrt += files('crt0.c')
+
+picocrt_machines += [
+  {
+    'name': 'fvp',
+    'suffix': '-fvp',
+  },
+]
-- 
2.43.0

